go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/google/go-cmp v0.3.1 // indirect
 	github.com/hashicorp/go-cleanhttp v0.5.1
 	github.com/hashicorp/go-gcp-common v0.5.0
+	github.com/hashicorp/go-retryablehttp v0.6.7 // indirect
```

Comment:
For clarity can you document how you updated the dependency? (manual edit vs `go mod` command)

Reply:
`go get github.com/hashicorp/go-retryablehttp` then `go mod vendor`